src/renderer/components/Transaction/TransactionForm/TransactionFormTransfer.vue

```diff
@@ -195,6 +195,17 @@ export default {
   },
 
   mounted () {
+    // Note: we set this here and not in the data property so validation is triggered properly when fields get pre-populated
+    if (this.schema) {
+      this.form = {
```

Comment:
Nested properties must be updated by the `$set` method:

```
this.$set(this.form, 'amount', this.schema.amount)
```

https://vuejs.org/v2/guide/reactivity.html#Change-Detection-Caveats

Reply:
But since you are defining the whole object it is probably already reactive 🤔

Reply:
I can change it to this:

<img width="516" alt="schermafbeelding 2018-12-06 om 15 54 07" src="https://user-images.githubusercontent.com/35610748/49591618-358e3b80-f96f-11e8-860a-8cf2cdf7b6da.png">

Also gets rid of setting some properties again while they don't change 🤷‍♂️